src/pages/signup/components/Signup.tsx

```
@@ -1,20 +1,39 @@
+import { zodResolver } from '@hookform/resolvers/zod';
 import Button from '@common/Button';
 import { Input } from '@common/Input';
 import useMediaQuery from '@hook/useMediaQuery';
-import { useState } from 'react';
+import {
+  SignupFormValues,
+  SignupSchema,
+} from '@validation/signup/SignupSchema';
 
 interface SignupProps {
   onNext: () => void | undefined;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)